ingest/indexer/service/blockprocessor/block_updates_indexer_block_process_strategy.go

```diff
@@ -40,6 +40,12 @@ func (f *blockUpdatesIndexerBlockProcessStrategy) publishChangedPools(ctx types.
 
 	pools := blockPools.GetAll()
 
+	// TODO: move to config.
+	if len(pools) > 200 {
+		// TODO: add counter and alert
+		// Threshold is arbitrary but should be a good starting point
+	}
+
```

Comment:
We should generally add a Prometheus counter for the number of pools being ingester.

We can define alert at infra level